src/components/ItemCard.css

```diff
@@ -22,3 +22,21 @@
 .ItemCard__name {
   color: var(--color-icons);
 }
+
+.ItemCard__button {
+  background-color: var(--color-icons);
+  color: var(--color-button-text);
+  border: 2px hidden;
+  border-radius: 25px;
+  padding: 0.25rem 0.2rem;
+}
+
+.ItemCard--btnDiv {
```

Comment:
Hi Boris, what is btnDiv? you can probably name it better. :)

Reply:
good point, I overlooked it